File: WebSolution/UIApp/src/app/time-sheet/models/TimeSheet.ts
import { TimeSheetDetails } from "./timesheetdetails";


export class Timesheet{
    constructor(public id:number,
                public status:string,
                public timesheetDate:string,
                public statusChangedDate:string,
                public employeeId:number){}
                
    public timeSheetDetails:TimeSheetDetails[]=[]
}

File: Solution/WebSolution/Layered/TFLOBL/Entities/TimesheetMgmt/TimeSheet.cs
namespace Transflower.TFLPortal.TFLOBL.Entities.TimesheetMgmt;


public class Timesheet
{
    public int Id { get; set; }
    public string? Status { get; set; }
    public int EmployeeId { get; set; }
    public DateTime TimesheetDate { get; set; }
    public DateTime? StatusChangedDate { get; set; }   
}
